nb.js

```diff
@@ -286,10 +286,12 @@ yargs(hideBin(process.argv))
             }
           }
         })
+        .demandCommand()
+        .help()
```

Comment:
Weirdly, `.demandCommand` is that one that auto-prints help messages if the args don't fit while `.help` simply ensures that the program prints the help messages when you add `--help` args.